tests/pilot_benchmark/snakemake/README.md

```diff
@@ -0,0 +1,33 @@
+# Snakemake pipeline execution
```

Comment:
Would be good to have test data set, add this to repo and run pilot on test data (so that users can run the pilot benchmark as a test)

Reply:
As of now, Matthew generates the test data. I would then include this data instead of empty file pointers :)